### DISCUSSION TITLE
Option to always show a collections entry for movies, without using the filter option. Like you've with the official Plex app

### DISCUSSION THREAD

**Discussion by TRaSH-** [pannal/plex-for-kodi#152] [Ideas]
When I want to see the collections of all the movies I need to use the filter option first, my wife doesn't like that option and would like to see an option so you can select collections right away.
Like you've with the official Plex app

**Comment by pannal** [comment]
So you want to see collections within the Movie view of a library?

**Reply by mvanbaak** [reply]
[edit]: found the feature request for exactly what I showed in this comment. Sorry for the noise. I was replying with the idea specified in https://github.com/pannal/plex-for-kodi/discussions/113 So I removed this reply.

**Comment by TRaSH-** [comment]
Yes like you would have with the normal plex client on the shield, this way you select your movie library or your collections list so you can see and choose a collection to watch or show
Don't have access to my shield at the moment, but with the web view it looks like this
![image](https://github.com/user-attachments/assets/038a5e46-482d-4f89-8060-83fb2af1f42b)

**Reply by bowlingbeeg** [reply]
It sounds like you're asking for the collections view which already exists.  Is that true or am I not understanding what you're asking for.

![image](https://github.com/user-attachments/assets/ddb8ccd7-6447-4218-88eb-201d0885fe41)

**Reply by pannal** [reply]
I think he wants the tabbed layout instead of having to open a dropdown

**Reply by TRaSH-** [reply]
Yes that's exactly what i want, i don't want to use the extra pull down menu to be able to access the collections, i want to be want able to choose it directly like you see with the original Plex app
